MediaBrowser.Model/Lyrics/LyricModel.cs

```diff
@@ -1,12 +1,11 @@
-using System;
 using System.Collections.Generic;
 
-namespace MediaBrowser.Controller.Lyrics;
+namespace MediaBrowser.Model.Lyrics;
 
 /// <summary>
 /// LyricResponse model.
 /// </summary>
-public class LyricResponse
+public class LyricModel
```

Comment:
```suggestion
public class LyricDto
```

This would make sense for the API part but might not make much sense for the other usages